apps/next-evans/components/pages/explore/RegionBlock.tsx

```diff
@@ -5,7 +5,9 @@ import BookSlider from './BookSlider';
 import MapSlider from './MapSlider';
 
 interface Props {
-  books: (Omit<DocumentWithMeta, 'numPages' | 'size'> & { region: Region })[];
+  books: (Omit<DocumentWithMeta, 'numPages' | 'size' | 'featuredDescription'> & {
+    region: Region;
+  })[];
```

Comment:
can you convert this to use `Array<T>` instead of `T[]`? i'm finding this type a bit hard to parse mentally.